ui/views_test.py

```diff
@@ -361,6 +361,20 @@ def test_program_thumbnail(self):
         resp = self.client.get('/')
         self.assertContains(resp, image.get_rendition('fill-345x265').url)
 
+    def test_courses_enrollment_text(self):
+        """Verify get_course_enrollment_test is called with each course in the program"""
+        course = CourseFactory(program=self.program_page.program)
+        course.save()
+        with patch('cms.models.get_course_enrollment_text') as get_enrollment_text:
+
+            self.client.get(self.program_page.url)
+            for course in self.program_page.program.course_set.all():
```

Comment:
Can we assert the order of the courses here using [call_args_list](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args_list)? We should also order it by `position_in_program` here and in `cms/models.py`